glusterd2/volgen2/generate.go

```diff
@@ -37,7 +37,7 @@ func generateClusterLevelVolfiles(clusterinfo []*volume.Volinfo, xopts *map[stri
 				if err != nil {
 					return err
 				}
-				err = save(peerid.String()+"-"+volfile.FileName, volfiledata)
+				err = save(peerid.String()+"-"+volfile.FileName, volfiledata, "-")
```

Comment:
Got it. I need to regenerate Cluster Volfiles

Reply:
After creating new issue https://github.com/gluster/glusterd2/issues/1005, I realized that we don't need to re-generate cluster level volfiles since volfiles are re-generated after every volume stop.

@rafikc30 Do you see any issues here?

Reply:
Yea, Once we modify volfile regenerate code for volume stop, then we don't need to worry about deleting the cluster level volfiles. We have to make sure that we do the modification for the last volume correctly, either we can delete the volfile here, or we can keep the first xlator.